homeassistant/components/climate/eq3btsmart.py

```diff
@@ -79,7 +76,7 @@ def __init__(self, _mac, _name):
     @property
     def available(self) -> bool:
         """Return if thermostat is available."""
-        return self.current_operation != STATE_UNKNOWN
+        return self.current_operation
```

Comment:
should be better code convention with `return self.current_operation is not None`